content/posts/jellyfin-10-6-0.md

```diff
@@ -0,0 +1,114 @@
+
+Further changes to the server include fixes for collection metadata issues ([jf#3117](https://github.com/jellyfin/jellyfin/pull/3117)), improvements to the metadata providers ([jf#3071](https://github.com/jellyfin/jellyfin/pull/3071), [jf#3056](https://github.com/jellyfin/jellyfin/pull/3056), [jf#3289](https://github.com/jellyfin/jellyfin/pull/3289)), and more core providers moved to plugins ([jf#3208](https://github.com/jellyfin/jellyfin/pull/3208)).
+
+# More web client improvements
+
+[Itegulov](https://github.com/itegulov) and [dkanada](https://github.com/dkanada/) have improved support for ebooks by adding an EPUB reader based on [epub.js](https://github.com/futurepress/epub.js/) ([jf-web#1263](https://github.com/jellyfin/jellyfin-web/pull/1263)). Reader support for more formats is in progress for future versions, including CBZ/CBR and PDF.
+
+As we used the new 10.5 details screen over the last few months, we noticed some improvements we could make to the experience. Team member [MrTimscampi](https://github.com/MrTimscampi), along with input from the rest of the web team and some of our users, did a second pass on that screen ([jf-web#949](https://github.com/jellyfin/jellyfin-web/pull/949), [jf-web#1206](https://github.com/jellyfin/jellyfin-web/pull/1206)), tightening the design and cleaning some visual issues along the way. [Delgan](https://github.com/jellyfin/jellyfin-web/pull/1406) put in the final touched to the page by avoiding a jump in the content when rendering the track selectors on the page ([jf-web#1406](https://github.com/jellyfin/jellyfin-web/pull/1406))
+
+<div class="juxtapose">
+    <img data-label="10.5.5" src="/images/posts/jellyfin-10-6-0/details-10-5.png" name="" />
+    <img data-label="10.6.0" src="/images/posts/jellyfin-10-6-0/details-10-6.png" name="" />
+</div>
+
+Team member [ferferga](https://github.com/ferferga) and contributor [samuel9554](https://github.com/samuel9554) have been working on redesigning our music experience. For 10.6, they overhauled our mobile music player interface and made significant changes to the mini player and the remote player UI ([jf-web#1056](https://github.com/jellyfin/jellyfin-web/pull/1056), [jf-web#1430](https://github.com/jellyfin/jellyfin-web/pull/1430)).
+
+<div class="juxtapose">
+    <img data-label="10.5.5" src="/images/posts/jellyfin-10-6-0/player-10-5.png" name="" />
+    <img data-label="10.6.0" src="/images/posts/jellyfin-10-6-0/player-10-6.png" name="" />
```

Comment:
would the images need ~/ at the beginning? the docs site has that issue

Reply:
It was working as-is for the 10.5 release post, so I think we should be good?